pkg/sql/schemachanger/scbuild/internal/scbuildstmt/helpers.go

```diff
@@ -990,8 +990,19 @@ func panicIfSchemaChangeIsDisallowed(tableElements ElementResultSet, n tree.Stat
 		panic(sqlerrors.NewSchemaChangeOnLockedTableErr(ns.Name))
 	}
 
+	var virtualColNames []string
```

Comment:
this could also get gated behind a `if ldrJobIds != nil` branch

Reply:
done